src/main/cpp/io/IntakeIO.cpp

```diff
@@ -0,0 +1,65 @@
+#include "io/IntakeIO.h"
+#include "Parameters.h"
+
```

Comment:
the replacement for the claw. we wrote it in a separate spot in case we needed to swap back to the claw at competition. I think it's safe to delete the old claw code now though.